terraform/environments/analytical-platform-compute/environment-configuration.tf

```diff
@@ -131,6 +132,11 @@ locals {
 
       /* UI */
       ui_hostname = "analytical-platform.service.justice.gov.uk"
+
+      /* LF Domain Tags */
+      cadet_lf_tags = {
+        domain = ["bold", "civil", "courts", "general", "criminal_history", "development_sandpit", "electronic_monitoring", "finance", "interventions", "opg", "performance", "risk", "people", "prison", "probation", "victims", "victims_case_management"] # extracted from bucket paths
```

Comment:
Can this be done programatically?

Reply:
Yes but I'll need to recreate most of my scanning script to achieve it and according to JHP, they change infrequently enough that hard-coding them is fine.

I wanted them here rather the database-access terraform because it makes it a lot more visible: these tags are created in this account, we should know where they're managed.

Reply:
⛏️  Personal preference, I would prefer this as a list (`domain_tags`, and then use in combination with `tag key = "domain"`, rather than leaving it as a map of a single key:value pair, as it could lead to overloading the `domain` [Lake Formation resources](https://github.com/ministryofjustice/modernisation-platform-environments/pull/8923/files#diff-471467f13a6c02134cff21bc098525baf070172dd4954a9610df37854d4b49e2R24) if anther key pair gets added to this map.